package.json

```diff
@@ -1,18 +1,21 @@
 {
   "name": "@burdaforward/composable-rules",
-  "version": "3.0.0",
+  "version": "3.0.1",
   "description": "A library for composable rule logic. Easily combine simple rules into more complex ones in a maintainable way using your custom logic and data.",
-  "unpkg": "./dist/umd/index.js",
-  "main": "./dist/cjs/index.js",
-  "types": "./dist/types/index.d.ts",
-  "module": "./dist/esm/index.js",
+  "main": "./dist/index.cjs",
+  "unpkg": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "exports": {
-    "require": "./dist/cjs/index.js",
-    "default": "./dist/esm/index.js"
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js",
+      "default": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    }
   },
   "scripts": {
-    "build": "npm run clean && tsc --build tsconfig.build.esm.json && tsc --build tsconfig.build.cjs.json && tsc --build tsconfig.build.umd.json",
-    "clean": "rm -rf dist",
+    "build": "tsup src/index.ts --format cjs,esm --dts --minify --clean",
```

Comment:
With `tsc` not being used anymore maybe you wanna add an extra script for typechecking and add it to the pipline. Like `npm run type-check` which does `tsc --noEmit`.

Tsup does not do any typechecks, just transpiling.